docs/testing.md

```diff
@@ -1,7 +1,8 @@
-CRI-Containerd Testing Guide
-============================
-This document assumes you have already setup the development environment (go, git, `cri-containerd` repo etc.).
- 
+========================
```

Comment:
Remove this line. Preview the document you'll see that.

Reply:
done